<?xml version="1.0" ?>
<launch>
    <arg name="gazebo"      default="true"  doc="Should the gazebo simulation be launched? Use false in case if you want to include this file and launch gazebo yourself" />
    <arg name="headless"    default="false" doc="Should the gazebo GUI be launched?" />
    <arg name="paused"      default="false" doc="Should the simulation directly be stopped at 0s?" />
    <arg name="world"       default="worlds/empty.world" doc="Filename to a SDF World for gazebo to use" />
    <arg name="rviz"        default="true" doc="Should RVIz be launched?" />

    <arg name="arm_id"      default="panda" doc="Name of the panda robot to spawn" />
    <arg name="use_gripper" default="true"  doc="Should a franka hand be mounted on the flange?" />
    <arg name="controller"  default="effector_cartesian_controller"     doc="Which example controller should be started? (One of {cartesian_impedance,model,force,joint_position,joint_velocity}_example_controller)" />

    <rosparam file="$(find panda_arm_control)/config/sim_controllers.yaml" subst_value="true" />
    <!--rosparam file="$(find panda_arm_control)/config/franka_hw_sim.yaml" subst_value="true" /-->

    <include file="$(find panda_arm_control)/launch/panda.launch" pass_all_args="true">
        <arg name="gazebo"      value="$(arg gazebo)" />
        <arg name="headless"    value="$(arg headless)" />
        <arg name="paused"      value="$(arg paused)" />
        <arg name="world"       value="$(arg world)" />
        <arg name="rviz"        value="$(arg rviz)" />

        <arg name="arm_id"      value="$(arg arm_id)" />
        <arg name="use_gripper" value="$(arg use_gripper)" />
        <arg name="controller"  value="$(arg controller)" />
    </include>

</launch>